rtengine/jdatasrc.cc

```diff
@@ -3,6 +3,13 @@
 #include <jerror.h>
 #include "jpeg.h"
 
+/*
+ * HA 2021-03-12
+ * Modification according to
+ * https://github.com/Beep6581/RawTherapee/commit/be67261d0bdeb21ffe15fe968dd5bc16edd407e0
+ * because otherwise it won't compile.
+ */
+  
 /*
```

Comment:
Where are the changes in this file coming from? These seem irrelevant to your PR.

Reply:
They are. But without them, it is apparently not possible to compile RT under Windows.

Reply:
But the commit you refer to clearly shows these changes have been part of the codebase since October last year. You probably branched off earlier than that. This is not a problem per se, just inconvenient because it introduces merge conflicts. However, adding these comments to `jdatasrc.cc` makes no sense, they are completely irrelevant to include in the final code.

Reply:
Yes, releases branch is definitely older than that. That one, then, should also be resolved by a rebasing.